scripts/build_brain.py

```
@@ -0,0 +1,711 @@
+            "cluster_count": len(clusters),
+        },
+        "nodes": [n.to_dict() for n in nodes],
+        "edges": edges,
+        "clusters": [{"id": c.id, "name": c.name, "count": c.count, "members": c.members, 
+                      "centroid": {"x": c.centroid[0], "y": c.centroid[1], "z": c.centroid[2]}} 
+                     for c in clusters],
+        "similarity_pairs": similarity_pairs,
+        "file_tree": data["tree"],
+    }
+    
+    with open(output_path, "w") as f:
```

Ruff (0.14.10)

Undefined name `output_path`

(F821)